mycroft/audio/speech.py

```diff
@@ -44,6 +44,14 @@ def handle_speak(event):
     Configuration.set_config_update_handlers(bus)
     global _last_stop_signal
 
+    # if the message is targeted and audio is not the target don't
+    # don't synthezise speech
+    event.context = event.context or {}
+    if event.context.get('destination') and \
+            ('debug_cli' in event.context['destination'] or
+             'audio' in event.context['destination']):
```

Comment:
should this be
```python
    if event.context.get('destination') and \
            ('debug_cli' not in event.context['destination'] or
             'audio' not in event.context['destination']):
```

That change seems to make the PR work for me.

Reply:
actually it should be 

```python
if event.context.get('destination') and not \
            ('debug_cli' in event.context['destination'] or
             'audio' in event.context['destination']):
```

We want TTS to happen when "debug_cli" is destination and also when "audio" is destination (and on multicasts when destination is missing)